proposals/0001-dep-process.md

```diff
@@ -124,7 +124,7 @@ new DEP is:
 DEPs should have a type:
 
 * **Standard** for technical changes to the protocol, on-disk formats, or
-  public APIs. These are intended to be *proscriptive*, and to clearly
+  public APIs. These are intended to be *prescriptive*, and to clearly
```

Comment:
"proscriptive" is defined as:

> the act of proscribing.

Where "proscribing" is defined as:

> to denounce or condemn (a thing) as dangerous or harmful; prohibit.

Which I figured wasn't the intended meaning, instead "prescriptive" seems to be a better fit here.